detector/aws_db_instance_duplicate_identifier_test.go

```diff
@@ -0,0 +1,175 @@
+                            "copy_tags_to_snapshot": "false",
+                            "db_subnet_group_name": "dbsubnet",
+                            "endpoint": "my-db.hogehoge.us-east-1.rds.amazonaws.com:3306",
+                            "engine": "mysql",
+                            "engine_version": "5.6.27",
+                            "hosted_zone_id": "HAND737292A",
+                            "id": "my-db",
+                            "identifier": "my-db",
+                            "instance_class": "db.t1.micro",
+                            "iops": "0",
+                            "kms_key_id": "",
+                            "license_model": "general-public-license",
+                            "maintenance_window": "mon:17:06-mon:17:36",
+                            "monitoring_interval": "0",
+                            "multi_az": "false",
+                            "name": "mydb",
+                            "option_group_name": "default:mysql-5-6",
+                            "parameter_group_name": "default.mysql5.6",
+                            "password": "password",
+                            "port": "3306",
+                            "publicly_accessible": "false",
+                            "replicas.#": "0",
+                            "replicate_source_db": "",
+                            "security_group_names.#": "0",
+                            "skip_final_snapshot": "true",
+                            "status": "available",
+                            "storage_encrypted": "false",
+                            "storage_type": "standard",
+                            "tags.%": "0",
+                            "timezone": "",
+                            "username": "foo",
+                            "vpc_security_group_ids.#": "1",
+                            "vpc_security_group_ids.3963419045": "sg-1234abcd"
+                        }
+                    },
+                    "provider": ""
+                }
+            }
+        }
+    ]
+}
+`,
+			Response: []*rds.DBInstance{
+				&rds.DBInstance{
+					DBInstanceIdentifier: aws.String("my-db"),
+				},
+				&rds.DBInstance{
+					DBInstanceIdentifier: aws.String("your-db"),
+				},
+			},
+			Issues: []*issue.Issue{},
+		},
+	}
+
+	for _, tc := range cases {
+		c := config.Init()
+		c.DeepCheck = true
+
+		awsClient := c.NewAwsClient()
+		ctrl := gomock.NewController(t)
+		defer ctrl.Finish()
+		rdsmock := mock.NewMockRDSAPI(ctrl)
+		rdsmock.EXPECT().DescribeDBInstances(&rds.DescribeDBInstancesInput{}).Return(&rds.DescribeDBInstancesOutput{
+			DBInstances: tc.Response,
+		}, nil)
+		awsClient.Rds = rdsmock
+
+		var issues = []*issue.Issue{}
+		TestDetectByCreatorName(
+			"CreateAwsDBInstanceDuplicateIdentifierDetector",
+			tc.Src,
+			tc.State,
+			c,
+			awsClient,
+			&issues,
+		)
+
+		if !reflect.DeepEqual(issues, tc.Issues) {
+			t.Fatalf("Bad: %s\nExpected: %s\n\ntestcase: %s", issues, tc.Issues, tc.Name)
```

Comment:
__[gometalinter]__
* [vet] arg issues for printf verb %s of wrong type: []*github.com/wata727/tflint/issue.Issue

<!-- Comment by [SideCI](https://sideci.com) -->